.eslintrc

```diff
@@ -20,14 +20,17 @@
     "max-len": [
       "warn",
       {
-        "code": 150,
+        "code": 350,
         "tabWidth": 2,
         "ignoreComments": true
       }
     ],
-    "react/prop-types": "error",
+    "react/prop-types": "off",
```

Comment:
this rule is pretty easy to follow, and adheres to better practices, I'd rather us not turn this off. if you're having trouble with router history/location, theres actually a node module that helps with prop-types for this. I'm implementing in a file I'm currently working on. lets chat to see why you want to disable this, and I can show you how i'm doing my prop-types.

Reply:
updating